libs/ledger-live-common/src/device-core/capabilities/isCustomLockScreenSupported.ts

```diff
@@ -0,0 +1,15 @@
+import { DeviceModelId } from "@ledgerhq/devices";
+
+export function isCustomLockScreenSupported(deviceModelId: DeviceModelId) {
+  switch (deviceModelId) {
+    case DeviceModelId.stax:
```

Comment:
Maybe showing (`return true;`) explicitly the value is a good idea, e.g. if we want to enable / disable one, it's easier and more clear.